src/main/java/online/talkandtravel/facade/impl/AuthenticationFacadeImpl.java

```diff
@@ -29,15 +33,23 @@ public class AuthenticationFacadeImpl implements AuthenticationFacade {
   private final TokenService tokenService;
   private final AuthenticationService authenticationService;
   private final UserMapper userMapper;
+  private final UserRepository userRepository;
```

Comment:
remomove UserRepository and UserMapper. Add UserService. Make additional methods if needed